packages/grafana-ui/src/components/SelectNG/SelectNG.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import { SelectNG } from './SelectNG';
+import { SelectableValue } from '@grafana/data';
+import { selectors } from '@grafana/e2e-selectors';
+import { act } from '@testing-library/react-hooks';
+
+const fakeOptions: Array<SelectableValue<string>> = [
+  {
+    value: 'opt1',
+    label: 'Option 1',
+  },
+  {
+    value: 'opt2',
+    label: 'Option 2',
+  },
+  {
+    value: 'opt3',
+    label: 'Option 3',
+  },
+];
+const fakeOnChangeHandler = () => {};
+describe('Select', () => {
+  describe('props', () => {
+    test('isOpen', async () => {
```

Comment:
Do we have use cases for `isOpen` to be controlled externally? Delegating this logic to Select would be better imo.